custom/icds_reports/tasks.py

```diff
@@ -1834,14 +1834,17 @@ def reconcile_data_not_in_ucr(reconciliation_status_pk):
             continue
         num_docs_retried += 1
         found_in_es = doc_id not in doc_ids_not_in_es
-        if not inserted_at or (sql_modified_on - inserted_at).seconds > 3600:
```

Comment:
This change looks slightly different than what you are talking about in the description. This isn't the 2s thing for time drift (that could produce false positives). This waits an hour after a change before deciding that it is missing, it should reduce false positives

Reply:
that's true but the logging was happening outside of this if statement, so the logging is now inside this if statement and the condition has changed.